docs/quickstart/cross-chain-transfers.md

```diff
@@ -0,0 +1,298 @@
+
+```js
+const privKey: string = "<YOUR-PRIVATE-KEY-HERE>";
+```
+
+##### Network Setting
+
+Depending on how you start your node, you might need to change the following variables in the code:
+
+
+```js
+const ip: string = "localhost";
+const port: number = 9650;
+const protocol: string = "http";
+const networkID: number = 5;
+```
+
+**IP Address**
+
+To use a local node started with [`--network-id=fuji`](../nodes/maintain/avalanchego-config-flags.md#network-id):
+
+```js
+const ip: string = "localhost";
+```
+
+Or to connect directly to the [Avalanche public server for Fuji Testnet](../apis/avalanchego/public-api-server.md) server:
+
+```js
+const ip: string = "api.avax-test.network";
+```
+
+**Port Number**
+
+To use a local node started with [`--network-id=fuji`](../nodes/maintain/avalanchego-config-flags.md#network-id):
+
+```js
+const port: number = 9650;
+```
+
+Or to connect directly to the [Avalanche public server for Fuji Testnet](../apis/avalanchego/public-api-server.md) server:
+
+```js
+const port: number = 443;
+```
+
+**Protocol**
+
+To use a local node started with [`--network-id=fuji`](../nodes/maintain/avalanchego-config-flags.md#network-id):
+
+```js
+const protocol: string = "http";
+```
+
+Or to connect directly to the [Avalanche public server for Fuji Testnet](../apis/avalanchego/public-api-server.md) server:
+
+```js
+const protocol: string = "https";
+```
+
+**Network ID**
+
+Depending on the networkID passed in when instantiating Avalanche, the encoded addresses used will have a distinctive Human Readable Part(HRP) per each network.
+
+_Example Address: 5 - X-`fuji`19rknw8l0grnfunjrzwxlxync6zrlu33yxqzg0h_
+
+For Fuji Testnet, 5 is the correct value to use.
+
+```js
+const networkID: number = 5;
+```
+
+To learn more  about  encoded addresses, click [here](../apis/avalanchejs/manage-x-chain-keys.md#encode-bech32-addresses).
+
+**Set the Correct Amount to Send:**
+
+By default the scripts send the wallet's entire AVAX balance:
+
+```js
+const balance: BN = new BN(getBalanceResponse.balance);
+const amount: BN = balance.sub(fee);
+```
+
+To send a different amount, please replace the code above with the following. Below sets a new value of 0.01 AVAX (10000000000000000 GWEI). Value is set in WEI format where 10 \*\* 18 WEI = 1 AVAX
+
+```js
+const value: BN = new BN("10000000000000000");
+const amount: BN = value.sub(fee);
+```
+
+:::tip
+Snowtrace provides a [unit converter](https://snowtrace.io/unitconverter) between different units
+:::
+
+
+#### Execute The Export And Import Scripts
+
+To transfer the specified amount token from X-Chain to C-Chain, the token needs to be first exported from the X-Chain, then imported to C-Chain. 
+
+
+Run the export script:
+```sh
+avalanchejs $ ts-node examples/avm/buildExportTx-cchain-avax.ts
+```
+
+This returns:
+```sh
+Success! TXID: Rgg2412kaczRYC3taasvG6bYoqG7tBQG6WfacNdumKDKsVWpF
+```
+
+
+Run the import script:
+```sh
+avalanchejs $ ts-node examples/evm/buildImportTx-xchain.ts
```

Comment:
Is there any changes needed in buildImportTx-xchain.ts before running it? Please list out all the changes and what should be done here.